pages/profiles/aas919.md

```diff
@@ -0,0 +1,17 @@
+# Ajinkya Shukla 
+Jersey City, NJ / EST
+Windows 7 Home
+
+<blockquote>
```

Comment:
@aas919 wonder for the seek of the exercise of writing markdown can we not use html tags

Reply:
@dogi @xyb994  I have moved to vi/profiles and changed to markdown. Please review the same.